### DISCUSSION TITLE
Online journal sync

### DISCUSSION THREAD

**Discussion by Toby-3-Strands** [jrnl-org/jrnl#1865] [Ideas]
In the long term I'd like to be able to interact with my journals through a mobile app. An intermediate step to achieve that would be to sync the journals to an online location.

I'm not proposing a dedicated jrnl server that everyone logs into. I'm suggesting that the user provides their own online store. It might be their Google Drive account or a git repo. There would be settings for this on a journal-by-journal basis. The user would specify the sync method then provide the details including authentication details. Then after every update to that journal, if there is a web connection the sync would be made.

Also there can be a command to resync without making changes.

**Comment by xeruf** [comment]
But if you use a filesharing service like Google Drive or in my case Syncthing, you can easily already do that by just syncing the folder where your journal is stored?
And then I edit my journal files from my phone in something like Logseq - not ideal, but works decently enough for now.
Down the line compatibility with some open-source journaling app (no need to write a new one I think) and org-journal would be very handy though.
